src/ui/components/Rating/index.js

```diff
@@ -86,19 +91,12 @@ export class RatingBase extends React.Component {
   }
 
   render() {
-    const { className, i18n, rating, readOnly, styleSize, isOwner } = this.props;
+    const { className, readOnly, styleSize, isOwner } = this.props;
     if (!RATING_STYLE_SIZES.includes(styleSize)) {
       throw new Error(
         `styleSize=${styleSize} is not a valid value; ` +
         `possible values: ${RATING_STYLE_SIZES.join(', ')}`);
     }
-    let description = i18n.gettext('This add-on has no ratings.');
```

Comment:
I don't think this stuff should be removed, except for `description = i18n.gettext('Click to rate this add-on');`

That could probably be changed to set `description = null` so that the "Rate this add X out of 5" is shown for `!readOnly` add-ons.